android/app/build.gradle

```diff
@@ -86,6 +86,8 @@ dependencies {
     implementation "io.data2viz.d2v:viz:0.8.0-RC10"
     implementation "io.data2viz.d2v:axis:0.8.0-RC10"
 
+    implementation 'com.crystal:crystalrangeseekbar:1.1.3'
+    implementation 'com.android.support.constraint:constraint-layout:1.1.3'
```

Comment:
don't repeat dependencies, constriant layout should be in the top of dependencies list